package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@foxglove/eslint-plugin",
-  "version": "0.22.1",
+  "version": "0.23.0",
```

Comment:
We typically do the version bump separately but I think for low-iteration repos like this one it might be alright.

Reply:
For this package I think not - but typically yes cause we might have other PRs we want to merge so no single PR gets to bump versions. Its nicer to feel comfortable reviewing a PR and separately making a decision about which version it will be.